pkg/common-controller/framework_test.go

```diff
@@ -904,18 +917,18 @@ func newSnapshot(
 			VolumeSnapshotContentName: &targetContentName,
 		}
 	}
-	if withFinalizer {
-		return withSnapshotFinalizer(&snapshot)
+	if withAllFinalizers {
+		return withSnapshotFinalizers([]*crdv1.VolumeSnapshot{&snapshot}, utils.VolumeSnapshotContentFinalizer, utils.VolumeSnapshotAsSourceFinalizer, utils.VolumeSnapshotBoundFinalizer)[0]
```

Comment:
withSnapshotFinalizers() only adds finalizers on VolumeSnapshot API object, right?  Why is utils.VolumeSnapshotContentFinalizer needed here?  utils.VolumeSnapshotContentFinalizer is added on VolumeSnapshotContent only.

Reply:
ok, removed the contentFinalizer.